src/components/navigation/NavigationBar.tsx

```diff
@@ -51,41 +47,50 @@ export default function NavigationBar(): ReactElement {
   return (
     <header id="navigation-header" className="navigation-header">
       <div className="navigation-header-container">
-        <Logo title="PromoPal" className="logo" height={50} />
+        <Link to={Paths.Home}>
+          <Logo title="PromoPal" className="logo" height={50} />
+        </Link>
         <Menu
           mode="horizontal"
           className="navigation-menu"
           onClick={({ key }) => setCurrent(key as Pages)}
           selectedKeys={[current]}
         >
-          <Menu.Item key={Pages.Home} className="navigation-menu-item" style={isActive(Pages.Home)}>
-            <Link to={Paths.Home}>Home</Link>
-          </Menu.Item>
```

Comment:
Not related to this code I highlighted, but the file itself

I kind of like it if `my account` is the very right, I feel like in websites its not common for your account details to be one of the main ones (b/c I read left to right for the tabs and left-er ones seem more important).
![image](https://user-images.githubusercontent.com/49849754/114284088-51ec8800-9a02-11eb-9082-64fa8f19eeb7.png)

Reply:
Oh yes you're right, didn't notice that good catch! I'll move this